src/features/flows/nRF9151/index.ts

```diff
@@ -206,6 +206,7 @@ const evaluationConfig = [
                 app: 'pc-nrfconnect-serial-terminal',
                 description:
                     'Use the Serial Terminal PC application as a serial interface to send AT commands to the device',
```

Comment:
```suggestion
                    'Use the Serial Terminal desktop application as a serial interface to send AT commands to the device',
```
Mac is not always considered a PC, so let's skip the PC ref.